src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 300ms linear;
```

Comment:
Better to create a variable for duration and apply both to header and catalog hovers